lib/screen/detail/artwork_detail_page.dart

```diff
@@ -550,6 +544,42 @@ class _ArtworkDetailPageState extends State<ArtworkDetailPage>
             },
           ),
         ],
+        OptionItem(
+          title: 'view_on_'.tr(args: [asset.secondaryMarketName]),
```

Comment:
please make it consistent with postcard. Just use "view on secondary market"

Reply:
the design specifies this actually